python/src/vmaf/__init__.py

```diff
@@ -1,13 +1,44 @@
 import os
+import subprocess
+
+try:
+    from matplotlib import pyplot as plt
```

Comment:
Importing `matplotlib.pyplot` only once, to avoid having to repeat this same pattern in multiple places in the code.

`matplot` has its quirks... they're better dealt with once like this